Sources/PrimerSDK/Classes/Core/PrimerHeadlessUniversalCheckout/Composable/FormWithRedirect/Ideal/IssuingBank.swift

```diff
@@ -6,15 +6,15 @@
 //
 
 import Foundation
-@objc public final class IssuingBank: NSObject {
+@objc public final class IssuingBank: NSObject, Encodable {
     public let id: String
     public let name: String
-    public let iconUrlStr: String?
+    public let iconUrl: String?
```

Comment:
Id rather we didn't make this change - as a user of the API I would expect `iconUrl` to be of type `URL`.
I would be happy with both leaving it as `iconUrlStr` or renaming to `iconUrlString`.

Reply:
Alright, i will change it to iconUrlString.

Reply:
@semirp this change was made for alignment with Android. 
Im happy to leave it as `iconUrl` here for alignment, but for complete correctness I think `iconUrlString` would be better 💅🏻

Reply:
Well, if that's better naming convention on iOS, you can change it. We can just map it differently on RN @NQuinn27